ONSdigital/response-operations-ui
# [TBL] Calls case service to generate enrolment code directly

# Motivation and Context

Removes the call to backstage for generating enrolment codes.

# What has changed



Changes the `generate_new_enrolment_code` method in reporting_units_controllers.py to hit the case service directly.

# How to test?



Run the unit tests, make sure they pass.
Tail the logs for backstage, case service. Hit the `/reporting-units/<ru_ref>/<collection_exercise_id>/new_enrolment_code` page. Make sure backstage doesn't get the call, but case service does.
